src/spryker/docker/image/console/root/lib/task/spryker/build.sh.twig

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+function task_spryker_build()
+{
+    {% if @('app.mode') == 'development' %}
+        passthru "composer install --no-interaction --dev --optimize-autoloader"
+    {% else %}
+        passthru "composer install --no-interaction --no-dev --optimize-autoloader"
+    {% endif %}
+        passthru "vendor/bin/install -r docker -s generate"
+        passthru "vendor/bin/install -r docker -s frontend"
+        passthru "vendor/bin/install -r docker -s cache"
+        # This will affect development mode
+        passthru "composer dumpautoload -a"
```

Comment:
still this to do, if it affects development then can be excluded from development

Reply:
this is what it does `Autoload classes from the classmap only`, meaning any new or renamed php classes don't get picked up automatically

Reply:
might still be best to run the dumpautoload (due to the install above potentially adding classes after the original --optimize-autoloader), but exclude the -a

Reply:
OK I get you, try now...

Reply:
https://github.com/inviqa/harness-base-php/pull/178/files/bce4cbfad89dda044d072e2d3cc7fcad26269027..999d484cd49a7c3b799b8e20110554b069659449